### DISCUSSION TITLE
[Proposal] Split season quests into multiple NPCs

### DISCUSSION THREAD

**Discussion by brian-man** [sunflower-land/sunflower-land#2543] [General]
(I've recommended this (multiple times) elsewhere but I figure it's probably good to make it a standalone proposal.)

It's probably problematic to introduce it for Dawn Breaker but please at least consider for next season.

# Context
For Solar Quest, the quest system was "known order, players discover, loops after 110".

(My understanding...)
For Dawn Breaker, there are three buckets -- Easy, Medium, Hard.  Quests are randomly chosen (per-farm) among Easy until all Easy quests are completed.  Then randomly chosen from Medium.  Then randomly from Hard.  It appears that once in the Hard bucket you just spin there for the rest of the season.

On the one hand, I think the season races are intended to be end-game for people that have expanded fully.
On the other hand, being completely stalled on a blocked quest is a terrible user experience (for new and average players).

# Proposal
Split the quest buckets into 3 NPCs - Easy NPC (Hank), Medium NPC (Elmer), Hard NPC (Budd).

- New players can spin on Easy.
- Average players can spin on Easy + Medium.
- Advanced players can spin on all 3.

**Comment by DeccoMx** [comment]
So, if i understand what you are saying, it's a way to have 3 Quests at the same time (for an Advanced players)?
Or when you finish Hank's quests you will then go with Elmer...?

**Reply by brian-man** [reply]
> So, if i understand what you are saying, it's a way to have 3 Quests at the same time (for an Advanced players)? Or when you finish Hank's quests you will then go with Elmer...?

The intent is to decouple the quest buckets so that players can advance without being blocked by a single bucket (typically medium or high) that has a quest that they don't have the requirements for (or don't want to do for whatever reason).

I think it's most interesting and useful if players can progress in the 3 buckets completely independently.  It would enhance the enjoyment and engagement, as well as the strategy.  As it currently sits, players get blocked or heavily delayed by single quests and the "gameplay" being promoted is "you can only do one task at a time on your farm" which is pretty poor gameplay IMO.  It's just not very compelling and often frustrating with the single quest model at a time model.

**Reply by alphacruze** [reply]
And certain quests are completely impossible for low level players to complete, even in the easy bracket

**Comment by Thebyber** [comment]
If they did that it would be to always do easy missions to get tickets faster... a mission to plant 20 kale gives me 3 tickets... for that I do 2 easy missions and I get those tickets in less time.

I like the proposal but they would have to upload the tickets for the difficult missions and limit the number of missions to do.

For example, if you make 50 tickets from the easy quests, you must make 50 tickets from the medium and hard levels to be able to do the easy quests again.

**Reply by brian-man** [reply]
> If they did that it would be to always do easy missions to get tickets faster... a mission to plant 20 kale gives me 3 tickets... for that I do 2 easy missions and I get those tickets in less time.
> 
> I like the proposal but they would have to upload the tickets for the difficult missions and limit the number of missions to do.
> 
> For example, if you make 50 tickets from the easy quests, you must make 50 tickets from the medium and hard levels to be able to do the easy quests again.

You're pointing out that tickets are not being awarded reasonably relative to quest difficulty and/or time.  That should be addressed for multiple reasons already but separately, IMO.

**Reply by Thebyber** [reply]
Of course.. currently it is like that. For your proposal to be attractive they should level that and control what I mentioned above!

**Comment by alphacruze** [comment]
Made a suggestion that could coincide well with this one:

[https://github.com/sunflower-land/sunflower-land/discussions/2544#discussion-5167648](2544)

**Comment by adamhannigan** [comment]
@brian-man Thanks for raising this and thanks everyone for the feedback ❤️ 

This is definitely aligned with the team's thinking. When we launch the plaza, we will be looking at splitting quests amongst different NPCs which will reward players with the tickets.

It will incorporate a bit of dialogue, lore + challenge. A player will be able to choose quests that suit their style of gameplay - however, it will still be the one's who grind the most that can earn the most tickets.
